pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -296,17 +297,21 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateClusterConfig(cephFilesyst
 		},
 	}
 
+	csiHostNetworkEnabled, err := strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_HOST_NETWORK", "false"))
+	if err != nil {
+		return errors.Wrap(err, "failed to parse value for 'CSI_ENABLE_HOST_NETWORK'")
+	}
```

Comment:
are we not adrressing fixing `netNamespaceFilePath` with this PR?

Reply:
Yes, when `CSI_ENABLE_HOST_NETWORK` is set to false the `netNamespaceFilePath` is updated to all clusterIDs and removed from all clusterIDs when `CSI_ENABLE_HOST_NETWORK` is set to true.